frontend/javascripts/oxalis/controller/camera_controller.ts

```diff
@@ -72,20 +72,25 @@ class CameraController extends React.PureComponent<Props> {
   storePropertyUnsubscribers: Array<(...args: Array<any>) => any>;
 
   componentDidMount() {
-    const far = 8000000;
-
-    for (const cam of _.values(this.props.cameras)) {
-      cam.near = 0;
-      cam.far = far;
-    }
     // Take the whole diagonal extent of the dataset to get the possible maximum extent of the dataset.
     // This is used as an indication to set the far plane. This needs to be multiplied by 2
     // as the dataset planes in the 3d viewport are offset by the maximum of width, height and extent to ensure the dataset is visible.
-    const datasetExtent = getDatasetExtentInLength(Store.getState().dataset);
+    const datasetExtent = getDatasetExtentInDatasourceUnit(Store.getState().dataset);
+    console.log(
+      "CameraController.componentDidMount: getDatasetExtentInDatasourceUnit",
+      datasetExtent,
+    );
     const diagonalDatasetExtent = Math.sqrt(
       datasetExtent.width ** 2 + datasetExtent.height ** 2 + datasetExtent.depth ** 2,
     );
-    this.props.cameras[OrthoViews.TDView].far = diagonalDatasetExtent * 2;
+    const far = Math.max(8000000, diagonalDatasetExtent * 2);
+
+    for (const cam of _.values(this.props.cameras)) {
+      cam.near = 0;
+      cam.far = far;
+    }
+    // This value is correct
+    this.props.cameras[OrthoViews.TDView].far = far;
```

Comment:
Previously, the value calculated for the far plane was overwritten in lines 103-108 after the 3d viewport was properly initialized. I modified the code to the larger of both values is kept.